www/_layouts/default.html

```diff
@@ -48,6 +48,7 @@
             <li><a href="/tournaments/">Tournaments</a></li>
             <li><a href="/skins/">Skin Database</a></li>
             <li><a href="/stats/">Statistics</a></li>
+            <li><a href="/libtw2-doc/">Documentation</a></li>
```

Comment:
I don't think we want to link it here and in the file below. After all this is not "DDNet Documentation", but "libtw2 Documentation"

Reply:
Yeah, wasn't sure about this. Do you have a better place to link the doc or should I just not link it for now?

Reply:
I'm not sure. It seems mostly relevant for developers.